### PR TITLE
Bank reconcile

### PR DESCRIPTION
**Pull request by ecino**
Migrate module account_reconcile_compassion to work with new modules in v8.

### CODE REVIEW

File: account_reconcile_compassion/wizard/reconcile_fund_wizard.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    fund_id = fields.Many2one(
+        'product.product', 'Fund', required=True,
+        default=lambda self: self._get_general_fund())
+    contract_ids = fields.Many2many(
+        'recurring.contract', compute='_get_contract_ids', store=True,
+        string='Related contracts',
+        default=lambda self: self._get_contract_ids())
+
+    def _get_contract_ids(self):
```
Comment (steveferry) [inline account_reconcile_compassion/wizard/reconcile_fund_wizard.py]: `set_contract_ids` ?

Comment (ecino) [inline account_reconcile_compassion/wizard/reconcile_fund_wizard.py]: Technically, it's both set and get :)
I think I prefer to keep this name.

File: account_reconcile_compassion/wizard/change_attribution_wizard.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    ##########################################################################
+    #                                 FIELDS                                 #
+    ##########################################################################
+    state = fields.Selection([
+        ('unrec', _('Unreconcile')),
+        ('change', _('Change Lines')),
+        ('rec', _('Reconcile'))], 'state', default='unrec')
+    invoice_line_ids = fields.Many2many(
+        'account.invoice.line', 'change_attribution_wizard_line_rel',
+        compute='_get_invoice_lines',
```
Comment (ndtran) [inline account_reconcile_compassion/wizard/change_attribution_wizard.py]: do we really need a compute as default already compute it ?

File: account_reconcile_compassion/wizard/reconcile_fund_wizard.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    """wizard that helps the user doing a full reconciliation when a customer
+    paid more than excepted. It puts the extra amount in a fund selected
+    in the self and fully reconcile the credit line. """
+    _name = 'reconcile.fund.wizard'
+
+    fund_id = fields.Many2one(
+        'product.product', 'Fund', required=True,
+        default=lambda self: self._get_general_fund())
+    contract_ids = fields.Many2many(
+        'recurring.contract', compute='_get_contract_ids', store=True,
```
Comment (ndtran) [inline account_reconcile_compassion/wizard/reconcile_fund_wizard.py]: do we really need a compute as default already compute it ?

File: account_reconcile_compassion/wizard/reconcile_split_payment_wizard.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+class reconcile_split_payment_wizard(models.TransientModel):
+    """Wizard that helps the user doing a full reconciliation when a customer
+    paid more than excepted. It splits the payment into two move lines so
+    that one invoice can be reconciled and the extra amount is kept in
+    the customer balance. """
+    _name = 'reconcile.split.payment.wizard'
+
+    comment = fields.Char('Indications on left amount', size=64)
+    contract_ids = fields.Many2many(
+        'recurring.contract', compute='_get_contract_ids', store=True,
```
Comment (ndtran) [inline account_reconcile_compassion/wizard/reconcile_split_payment_wizard.py]: do we really need a compute as default already compute it ?

File: account_reconcile_compassion/wizard/change_attribution_wizard.py (hunk 2 above)
Comment (ecino) [inline account_reconcile_compassion/wizard/change_attribution_wizard.py]: good remark, I will check if it works without compute.

**Comment by ecino** [comment]
Corrections made.

**Pull request merged**